drivers/hmis_data_quality_tool/app/models/hmis_data_quality_tool/enrollment.rb

```diff
@@ -439,6 +489,10 @@ def report_item_fields_from_enrollment(report_items:, enrollment:, report:) # ru
       false
     end
 
+    def self.adult?(item)
```

Comment:
If we're going to define this, let's use it in the `hoh_or_adult?` method just below.

Reply:
will update `hoh_or_adult?` to use this in the next commit